rurusetto/wiki/templates/wiki/edit_ruleset_wiki.html

```diff
@@ -68,6 +68,17 @@ <h1>Icon & Logo</h1>
                     </div>
                   </div>
             </div>
+            <div class="fieldWrapper">
+              <div class="mb-3 row">
+                 <label class="col-sm-2 col-form-label form-label" for="id_icon">{{ form.dark_icon.label_tag }}</label>
+                 <div class="col-sm-10">
+                     <div class="raleway-font form-current">Current Dark Icon : <a href="{{ form.dark_icon.value.url }}">{{ form.dark_icon.value }}</a></div>
+                   <input type="file" name="icon" accept="image/*" id="id_icon" class="form-control">
```

Comment:
```suggestion
                   <input type="file" name="light_icon" accept="image/*" id="id_light_icon" class="form-control">
```
The ID of the form is wrong.